.prettierignore

```diff
@@ -3,3 +3,4 @@ dist
 node_modules
 /polaris-react/build
 /polaris-react/build-internal
+/polaris-react/src/styles/shared/_breakpoints.scss
```

Comment:
Temporarily ignoring the `_breakpoints.scss` file as the formatting was [making the Scss variables harder to read](https://github.com/Shopify/polaris/blob/437c28ef294e7948d73252a8eff2fedc1e2d5958/polaris-react/src/styles/shared/_breakpoints.scss#L28-L55).